### PR TITLE
added forget modal functionality to addresses

### PR DESCRIPTION
**Pull request by marnibrewster**

### CODE REVIEW

File: packages/app-addresses/src/Forgetting.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      </Modal.Actions>
+    );
+  }
+
+  renderContent () {
+    const { t } = this.props;
+    return [
+      <Modal.Header key='header'>
+        {t('forget.header', {
+          defaultValue: 'Are you sure you want to delete this account?'
```
[the remark this answers is not in the document]
Comment (jacogr) [reply packages/app-addresses/src/Forgetting.tsx]: I don't immediately have a better suggestion. Agree, it is not perfect. Let's see if something pops up.

**Comment by jacogr** [comment]
Looks great. I would just make the same title adjustments as recommended in #287 (i.e. trim the title, optionally add some explanation text if we want/need more hand-holding and clarity)

**Comment by jacogr** [comment]
Ummm... I saw that yesterday, it would seem the comments I made there was lost and/or not available showing. (Screenshot and all - really have no idea if it was a GH mishap or something else sinister.) My apologies, I am clueless as to why it is not showing. Will re-do.

**Comment by marnibrewster** [comment]
Thank you! I will work on this as soon as I see your comments! Hopefully not something I deleted by accident.

**Comment by marnibrewster** [comment]
Strange - hopefully not something I deleted on accident? Hopefully, this was the change you wanted, but if not, let me know and I will complete today.

**Comment by jacogr** [comment]
No, we found it - I was doing too much, so the commented added on another (unrelated) PR. Entirely my mistake.

**Pull request merged**